.buildkite/ftr_configs.yml

```diff
@@ -21,6 +21,7 @@ disabled:
   - x-pack/test/functional/config.edge.js
 
   # Cypress configs, for now these are still run manually
+  - x-pack/test/security_solution_cypress/cli_config_explore_parallel.ts
```

Comment:
I believe we should remove from this file the configurations we are deleting on this PR.

Reply:
NIT: Maintain the alphabetical order of the list.

Reply:
Great catch. Changes pushed!